File: ClientApp/src/app/model/question.ts
export interface Question {
  Title: string;

  // 1-单选选择题
  // 2-不定向选择
  // 3-填空题
  // 4-简答题
  Type: number;
  QuestionContent: string;
  Answer: String[];
  Analysis: String;
}

File: Model/Question.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebAngular.Model
{
    public class Question
    {
        public int Id { get; set; }
        public int Title { get; set; }
        public string Answer { get; set; }
        public string Content { get; set; }
        public int Type { get; set; }
        public string Analysis { get; set; }
    }
}
